app/src/main/java/app/revanced/manager/ui/viewmodel/InstallerViewModel.kt

```diff
@@ -276,6 +308,12 @@ class InstallerViewModel(
     }
 }
 
+internal data class PackageInstallerResult(
```

Comment:
```suggestion
data class PackageInstallerResult(
```
Is internal necessary?